cpuprofilenode.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Roger Chapman and the v8go contributors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package v8go
+
+// #include <stdlib.h>
+// #include "v8go.h"
+import "C"
+
+// CPUProfileNode represents a node in a call graph.
+type CPUProfileNode struct {
+	ptr C.CpuProfileNodePtr
+	iso *Isolate
+}
+
+// Returns function name (empty string for anonymous functions.)
+func (c *CPUProfileNode) GetFunctionName() string {
```

Comment:
I thought the intention was to have the Go API match the V8 API. In this case and others for these components, the v8 functions are called `GetSomething`. It felt unintuitive to not follow that convention since we are trying to map as closely to v8 where we can and make discovering docs for our own functions easy to find in v8 api docs.

https://github.com/rogchap/v8go/blob/ed9a5f0761e2d7ff86dc622fb742f35f570c905e/deps/include/v8-profiler.h#L86

Reply:
v8go is both trying to match the v8 API and provide an API that users of go would expect, which why I made the suggestion.  However, it does look like there are a few existing functions that use the `Get` prefix (GetFunction, GetPromise, GetHeapStatistics).  If we were to differ from v8 to make it more idiomatic for Go, we should be consistent about it.  So I think you are right in using the Get prefix